main-site/components/adventure-ahead-section/AdventureAheadSection.styles.ts

```diff
@@ -0,0 +1,45 @@
+import styled from 'styled-components';
+import { P } from '../../../shared-ui/style/typography';
+import { max } from '../../../shared-ui/lib/responsive';
+import { colors } from '../../../shared-ui/style/colors';
+
+const StyledAdventureAheadSectionContainer = styled.div`
+  display: flex;
+  padding-top: 16em;
```

Comment:
When merging main into your branch, make sure to change the padding accordingly so that it's a good amount and not too much in relation to the previous section